async handle(s) {
  const {  } = this;
  if (true) {
    const x = 2;
    if ((true && ((x + 2) === 4))) {
      if ((true && ((x + 3) === 4))) {
        {((cond) => {if (!cond) throw new Error(`assertion failed: 'false'`)})(false)};
      }
      else if ((true && ((x + 3) === 6))) {
        {((cond) => {if (!cond) throw new Error(`assertion failed: 'false'`)})(false)};
      }
      else if ((false || ((x + 3) === 5))) {
        {((cond) => {if (!cond) throw new Error(`assertion failed: 'true'`)})(true)};
      }
      else {
        {((cond) => {if (!cond) throw new Error(`assertion failed: 'false'`)})(false)};
      }
    }
    else {
      {((cond) => {if (!cond) throw new Error(`assertion failed: 'false'`)})(false)};
    }
  }
}
